src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -22,6 +22,7 @@ public SuperStructure(
 
   public Command levelFour(BooleanSupplier score) {
     return Commands.sequence(
+        Commands.print("Running L4"),
```

Comment:
#43 adds names for these, which removes the need for debug printing. So please delete this.